addons/edi_notifier/models/email_notifier.py

```diff
@@ -35,16 +39,24 @@ def _notify(self, notifier, event_type, recs):
                 template = template.with_context(attachments=attachments)
             email_values = None
             if attachments:
-                email_values = {'attachment_ids': attachments.mapped('id')}
+                email_values = {"attachment_ids": attachments.mapped("id")}
             template.send_mail(rec.id, force_send=True, email_values=email_values)
 
+    def check_disbaled_edi_notifier(self):
+        return config.get_misc("edi", "disable_edi_notifications", False)
```

Comment:
Does the story want notifications disabled or enabled by default?

Reply:
It is a bit ambiguous but the story says "I don't want notifiers to be enabled by default on my UAT instance" so I guess it just means they have to be enabled manually on the live instance (Rather than having to manually turn them off for UAT).

Reply:
I think here we can add notifiers on customer instances, we will make it based on a config field against the email_notifier object

Reply:
Yes, please. And follow the same behaviour as the other safety catches.